DataProcessing/data_tagger.py

```diff
@@ -0,0 +1,102 @@
+import os
+import pickle
+from collections import defaultdict
+from matplotlib import pyplot as plt
+
+
+# todo all of these actions need to be done to the db, currently they all run in O(n) for each Crop changed !
+from DataProcessing.dataProcessingConstants import ID_TO_NAME
+
+
+def mark_vague(db, track, crop_inds):
+    for crop_id in crop_inds:
+        db_index = db.index(track[crop_id])
+        db[db_index].is_vague = True
+
```

Comment:
Once we start using the DB we need to change all these functions to update the `Crop` object instead of the pickle and then save the object to the DB.